tests/test_cmdline.py

```diff
@@ -25,6 +26,15 @@ def test_cmdline_attch_extract(script_runner):
     shutil.rmtree(tmpdir)
 
 
+def test_cmdline_zip_extract(script_runner):
+    tmpdir = tempfile.mkdtemp()
+    ret = script_runner.run('tnefparse', '-z', '-p', tmpdir, 'tests/examples/one-file.tnef')
+    assert os.path.isfile(tmpdir + '/attachments.zip')
+    assert ret.stderr == 'Successfully wrote attachments.zip\n'
+    assert ret.success
+    shutil.rmtree(tmpdir)
+
```

Comment:
I think this test is sufficient!

Just in case we want to make sure a valid zip file gets created, I use something like this in my other projects test cases:
```
    zip_stream = io.BytesIO(bytes_)
    zip_file = zipfile.ZipFile(zip_stream)
    assert zip_file.namelist() == ['software.lic']
```

Reply:
added, thanks!